e2e/tests/oai/002-database.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   upstream:
     repo: oai-core-packages
     package: database
-    revision: r3
```

Comment:
@arora-sagar can this also be main? Previously it was r2 and I was wondering if that's what we want.

Reply:
Yeah sure let me merge